docs/features/rbd-snapshot-metadata.md

```diff
@@ -0,0 +1,134 @@
+# Guide for RBD Deployment with CSI Snapshot-Metadata Sidecar
+
+> ⚠️ **Warning - Developer Preview Feature**:
```

Comment:
please mentioned this as Alpha feature as that is the terminology we use in upstream , same change in required in line 4 as well